README.md

```diff
@@ -20,7 +20,7 @@
 
 ## About this Repository
 
-This repository contains the source files of the official Corona-Warn-App landing page as it is available at [e.coronawarn.app](https://e.coronawarn.app) (for event-registration) or at [s.coronawarn.app](https://s.coronawarn.app) (for rapid antigen tests). The page opens for first-time users who have not installed the CWA app after scanning a check-in QR code or a QR code for a rapid antigen test result with a native camera app. The page also opens for users who clicked a link on an external website and the browser does not redirect to the CWA. This might be due to preference settings of the browser app and/or OS settings.
+This repository contains the source files of the official Corona-Warn-App landing page as it is available at [e.coronawarn.app](https://e.coronawarn.app) (for event-registration),[s.coronawarn.app](https://s.coronawarn.app) (for rapid antigen tests) & [p.coronawarn.app](https://p.coronawarn.app) (for rapid PCR tests). The page opens for first-time users who have not installed the CWA app after scanning a check-in QR code or a QR code for a rapid antigen/PCR test result with a native camera app. The page also opens for users who clicked a link on an external website and the browser does not redirect to the CWA. This might be due to preference settings of the browser app and/or OS settings.
```

Comment:
What are rapid PCR tests? I think rapid here is c&w.

Reply:
I just have this information from here: https://github.com/corona-warn-app/cwa-app-ios/pull/4185

Reply:
I assume that "rapid PCR tests" are the same as "PoC-NAT-Tests".

"PoC-NAT-Tests sind Tests, die wie der PCR-Test auch auf der Nukleinsäureamplifikationstechnik (NAT) basieren, jedoch vor Ort (Point of Care, PoC), also patientennah und kurzfristig vor Ort ausgewertet werden können. Daher sind sie in der Anwendung nicht auf ein Labor angewiesen."

Taken from https://www.bundesgesundheitsministerium.de/coronavirus/nationale-teststrategie/faq-covid-19-tests.html#c20484.

Reply:
@thomasaugsten Could you give us an update reg. PoC-NAT-Tests / rapid PCR tests? Are any testing sites which offer such a test connected to the CWA? Will there be any connected in the near future?